S2VX.Game/Story/Note/EditorHoldNote.cs

```diff
@@ -16,12 +19,28 @@ public class EditorHoldNote : HoldNote {
         [Resolved]
         private S2VXStory Story { get; set; }
 
+        [Resolved]
+        private ScreenStack Screens { get; set; }
+
         [BackgroundDependencyLoader]
         private void Load(AudioManager audio) {
             Hit = audio.Samples.Get("hit");
             HitSoundTimes = new List<double>() { HitTime, EndTime };
+            HeadAnchor = new EditorHoldNoteAnchor(this, true);
```

Comment:
bump

Reply:
I don't know if I can do that easily since Approaches comes after Notes in S2VXStory.cs so I can't show the anchor stuff on top with Depth. I'd probably have to make anchor/anchorPath separate from EditorHoldNote to draw it after Approaches for it to show on top

Reply:
We can defer this to another task